nengo/tests/test_transforms_conv.py

```diff
@@ -44,6 +44,46 @@ def test_convolution_shape(padding, stride, k_size, x_mul, rng, allclose):
         assert allclose(y_np, y_tf)
 
 
+@pytest.mark.parametrize("groups", (1, 2))
+@pytest.mark.parametrize("x_mul", (1, 2))
+@pytest.mark.parametrize("k_size", (1, 2))
+@pytest.mark.parametrize("stride", (1, 2))
+@pytest.mark.parametrize("padding", ("same", "valid"))
+def test_convolution_group_shape(padding, stride, k_size, x_mul, groups, rng, allclose):
```

Comment:
Again, I would have a common helper function for both this and the test above, so that we're not duplicating code.